app/services/publisher_wallet_getter.rb

```diff
@@ -140,6 +147,36 @@ def total_balance_bat(accounts)
     accounts.map {|account| account["balance"].to_d }.reduce(0, :+)
   end
 
+  def last_settlement_balance(transactions)
```

Comment:
I'm not sure if it's better to have this method in the PublisherWalletGetter like it is now,or if I should add a `last_settlement_balance` boolean parameter to the PublisherTransactionsGetter, and have this be a private method in the PublisherTransactionsGetter instead.

Reply:
Oh I nitpicking and meant to say we could merge the two blocks below since you're iterating over the same array twice.